moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -582,8 +582,16 @@ def place(self, object_name, location=None, plan_only=False):
             result = self._g.place(object_name, conversions.pose_to_list(location), plan_only)
         elif type(location) is PlaceLocation:
             result = self._g.place(object_name, conversions.msg_to_string(location), plan_only)
+        elif type(location) is list:
```

Comment:
Would this also work for a tuple type?

Reply:
It won't as it is right now. To accept tuple types we should add the check there as `elif type(location) is list or type(location) is tuple`. I didn't think about the option of adding a tuple of locations instead of a list, but we can check that too if needed.